README.md

```diff
@@ -29,6 +29,8 @@ future we plan to make libmuser multi-threaded. The application can be
 implemented in whatever way is convenient, e.g. as a Python script using
 bindings, on the cloud, etc.
 
+IRC channel: ##muser on Freenode.
```

Comment:
Let's use #muser as that will be official. Based on a quick reading (I didn't know this :) ), ## on freenode is for non-official project channels on freenode.

Reply:
done